hack/components/bump-kubemacpool.sh

```diff
@@ -59,6 +59,15 @@ patches:
   target:
     version: v1
     kind: Namespace
+- path: add-pod-template-label-allow-access-cluster-services_patch.yaml
+  target:
+    version: v1
+    kind: Deployment
+- path: add-pod-template-label-allow-prometheus-access_patch.yaml
+  target:
+    version: v1
+    kind: Deployment
+    name: mac-controller-manager
```

Comment:
Are you sure you cannot inline the json patch with kubebuilder things instead of generating files and apply them ?

Reply:
kubebuilder does provide generating NetworkPolicy for a project webhooks and metric endpoint using markers [[1] ](https://github.com/kubernetes-sigs/kubebuilder/pull/3853/files)(the +kubebuilder annotation), although it seems documnation about it lacking, it requires feasibility check.
Unfortunately kubemapool repo doesnt make use of kubebulder, it use plain controller-gen, which doesnt provider this functionality.
Transitioning the project to use kubebuilder has cost, naturally. I hope we can do it later on.

Note that in the context of this repo, the automation takes the released kubemacpool installtion manifest, generates a template form it and embed the template in CNAO.
Such changes as generating the network-policy using other tooling should be done in kubemacpool repo.